CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - requirement test execution logging
 - test for glue:R0056
+- message encoding errors are summarized by default.
```

Comment:
add both configuration options to changelog

Reply:
Done.